heartbeat/docker-compose

```diff
@@ -36,7 +35,10 @@
 . ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
 
 # Defaults
-OCF_RESKEY_binpath_default=/usr/bin/docker-compose
+OCF_RESKEY_binpath_default=/usr/local/bin/docker-compose
```

Comment:
You could set this to "" (you shouldnt manipulate the _default values, and the 2nd line below wont ever run.

If you set default to "" and update the metadata to say it auto-detects when not specified you can add the following line after line 43: `[ -z "$OCF_RESKEY_binpath" ] && OCF_RESKEY_binpath="$(which docker-compose 2>/dev/null)"`